### PR TITLE
german localisation

### PR DESCRIPTION
**Pull request by iemg**
first version of german localisation, complete, but some phrases are maybe too long.

### CODE REVIEW

**Comment by SlamaFR** [comment]
Thank you for your contribution! Since I can't speak German, I used ChatGPT to check for informal tenses, here is its feedback:

> ### Informal Phrases:
> 1. **"Alles klar"** (in `browser_check.dismiss`) – This is informal and commonly used in casual conversations. A more formal alternative would be something like **"Verstanden"** or **"OK"**.
> 
> 3. **"Neues Projekt"** (in `new_project.accept`) – While not offensive, this is more direct. A slightly more formal option could be **"Neues Projekt beginnen"**.
> 
> 4. **"Später"** (in `new_update_available.later`) – While this is acceptable, **"Zu einem späteren Zeitpunkt"** might sound a bit more formal.
> 
> ### Other Suggestions:
> - **"Trotzdem fortfahren"** (in `outdated_project.reject`) could be softened to a more neutral phrase like **"Dennoch fortfahren"** for a less abrupt tone.
> 
> - **"Problembehandlung"** (in multiple places) is correct, but in a very technical or formal context, you might use **"Fehlerbehebung"** to be clearer about solving issues.

Feel free to check for yourself which suggestions are relevant 😉

**Comment by iemg** [comment]
Hi! Thanks for the input, that's really helpful for checking! I think it's mostly a question of taste. I would suggest to stay with my choices, mainly because of shortness (2, 3). "Trotzdem fortfahren" is by choice to have the attention before clicking away an error message. And "Fehlerbehandlung" – well, "Fehler" are bugs and "Problem" are troubles, so the latter might be more appropriate in my view. But if the tone feels not right while usage, we can still consider changing, would that be ok for you? :)

Merci !

**Comment by iemg** [comment]
(sorry, clicked the wrong button – lacking coffee today :D)

**Comment by SlamaFR** [comment]
Thank you for your answer!

So if I understand everything, "Verstanden" is a better choice than "Alles klar", "Fehlerbehebung" is better than "Problembehandlung". For the other points, your choices are better.

Am I right?

**Comment by iemg** [comment]
No, sorry, I've not written it clesr enough: I would keep all my choices :)

File: locales/de.json [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+        "header": "Haltestelle: Optionen",
+        "interest_point": "relevanter Ort",
+        "stop_name": "Name",
+        "subtitle": "Untertitel",
+        "terminus": "Endstation"
+      },
+      "troubleshooting": {
+        "github": "erstellen Sie ein Issue auf GitHub",
+        "header": "Problembehandlung",
+        "notice": "Folgende Informationen können bei Fehlern in BULB zur Problembehandlung beitragen. Sollten Sie einen Fehler entdecken, {github} oder {twitter}, en précisant les informations ci-dessous.",
```
Comment (SlamaFR) [inline locales/de.json:234]: Meh, just spotted some forgotten french here... If you could just give me the missing translation

Comment (iemg) [reply locales/de.json:234]: Oh sorry, I missed that 😣 That should be:`       "notice": "Folgende Informationen können bei Fehlern in BULB zur Problembehandlung beitragen. Sollten Sie einen Fehler entdecken, {github} oder {twitter}, und übermitteln Sie mit der Problembeschreibung die untenstehenden Informationen.",`

Comment (SlamaFR) [reply locales/de.json:234]: Thanks!

**Pull request merged**